### PR TITLE
Set Python version in Travis build

### LINKED ISSUE #24: Set Python version in Travis build
Currently the Travis build relies on the Python version shipped by Travis. Instead it should be defined for the build itself. For more information see [this documentation](https://docs.travis-ci.com/user/languages/python/#specifying-python-versions).

### PR DESCRIPTION
**Pull request by VictorArranz**
Resolves #24

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"  # 3.8 development branch
+  - "nightly"  # nightly build
```
Comment (VictorArranz) [inline .travis.yml:13]: @HorizonNet I have a question I want to clarify. Shall I add all of these python versions like in the link you shared to me or just some of them or a specific one?

Before these changes 3.6 version was used by default.

Comment (HorizonNet) [reply .travis.yml:13]: Please continue to use 3.6 for now. We don't need the other one's right now.

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,19 @@
 sudo: required
 
 language: python
+python:
+<<<<<<< HEAD
```
Comment (HorizonNet) [inline .travis.yml:6]: It seems that you tried to merge, but the merge commit markers are still included.

**Review by HorizonNet — CHANGES_REQUESTED** [review]
The merge seems to be messed up.

**Comment by VictorArranz** [comment]
I've got "File permissions unset or incorrect" error during the build. what are the permission shall I set?

I had a look to the template command docu I want to know what value I should set for these params:

    owner -> any or nothing?
    group -> any or nothing?
    mode -> Which one? We're are getting error because this one is not fullfill.

**Comment by VictorArranz** [comment]
@HorizonNet So far I've tried with 644 but did not work.

**Comment by HorizonNet** [comment]
@VictorArranz Where do you get this error? Looking at the Travis build the problem lies somewhere else.

**Comment by VictorArranz** [comment]
@HorizonNet sorry for my previous comment I was wrong, the error was solved using 644 permission, probably I checked the wrong Travis build. Now the error is another one. I will solve it.

**Comment by VictorArranz** [comment]
@HorizonNet This ticket is ready for review.

File: molecule/default/molecule.yml [hunk 3]
```diff
@@ -3,8 +3,11 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
+lint: |
+  set -e
+  yamllint .
+#  ansible-lint
```
Comment (HorizonNet) [inline molecule/default/molecule.yml:9]: If ansible-lint and flake8 are not used, please remove these lines.

**Review by HorizonNet — CHANGES_REQUESTED** [review]
Build and changes look good. Left a final point. Afterwards we should be ready to merge.

**Comment by VictorArranz** [comment]
@HorizonNet the requested change is addressed.

**Review by HorizonNet — APPROVED** [review]
LGTM. Let's get it in.

**Pull request merged**